packages/fiori-app-sub-generator/src/types/common.ts

```diff
@@ -21,15 +21,15 @@ export type WorkspaceFolder = {
     path: string;
 };
 
-type DeploySubGenPromptOptions = {
-    targetName: CommonPromptOptions;
-};
+export type DeploySubGenPromptOptions =
```

Comment:
Shouldnt this type be defined in `@sap-ux/deploy-config-sub-generator`. Anyway we should alrady have that dep since we `composeWith` from this gen?

Reply:
Yes I'd have preferred to all of this in `@sap-ux/deploy-config-sub-generator` but it wasn't easily doable due to circular dependencies.
`@sap-ux/@sap-ux/fiori-app-sub-generator` is already a dev dep in `@sap-ux/deploy-config-sub-generator` because of the `AppConfig` needed for headless.

Moving `AppConfig` into the shared module also comes with it's issues

Reply:
Should be able to resolve once https://github.com/SAP/open-ux-tools/pull/3466 goes in